test/integration/util.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const template = require('lodash.template');
+
+const fileName = 'test.js';
+const renderDir = path.join(__dirname, 'render');
+
+function getHTML(file) {
+    return file.replace('.js', '.html');
+}
+
+function getPNG(file) {
+    return file.replace('.js', '.png');
+}
+
+function getOutPNG(file) {
+    return file.replace('.js', '_out.png');
+}
+
+function getName(file) {
+    return file.substr(
+        renderDir.length,
+        file.length - renderDir.length - fileName.length - 1
+    );
+}
+
+function loadOptions() {
+    return {
+        delay: 100,
+        viewportWidth: 400,
+        viewportHeight: 300,
+        headless: process.platform === 'linux'
+        // waitForFn: () => window.mapLoaded
```

Comment:
cannot we use this event with geojson maps?

Reply:
It is no needed here, I added just for testing.